package.json

```
@@ -3,6 +3,7 @@
   "version": "0.1.7",
   "description": "Manipulate files with escalated privileges",
   "main": "index.js",
+  "typings": "index.d.ts",
```

It looks like the [convention for this is `types`](https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html), although `typings` will still work.

```suggestion
  "types": "index.d.ts",
```

---

Yeah, I tested this out with `npm pack` and then installed it into Desktop, but if `types` is the recommended field then I'm :thumbsup: to switching over.